packages/menu/src/testUtils/withMenuContextDecorator.testutils.tsx

```diff
@@ -15,11 +16,20 @@ import { MenuItem } from '../MenuItem';
 export const withMenuContext =
   (): InstanceDecorator<typeof MenuItem & typeof Menu> => (Instance, ctx) => {
     const {
-      args: { darkMode: darkModeProp, renderDarkMenu },
+      args: {
+        darkMode: darkModeProp,
+        renderDarkMenu,
+        highlight,
+        moveHighlight,
+        setHighlight,
+      },
     } = ctx ?? {
       args: {
         darkMode: false,
         renderDarkMenu: false,
+        highlight: undefined,
+        moveHighlight: noop,
```

Comment:
What's the diff between `noop` and `undefined`?

Reply:
`moveHighlight` has to be defined, so here we use `noop`, which is a function that returns undefined